### ISSUE TITLE
Allow annotations for wg-access-server-web service in Helm

### ISSUE THREAD

**Issue by hongkongkiwi** [Place1/wg-access-server#81]
I would like to add some additional annotations to the wg-access-server-web service when installing via helm.

I can add annotations to the wireguard service, but the web service seems to have this field missing.

Could you please add it?

**Comment by hongkongkiwi** [comment]
I realised you already have this, but there is a small bug in the indentation in the deploy/helm/template/service.yaml file. It needs to be updated to:
```
{{- if .Values.web.service.annotations }}
  annotations:
{{ toYaml .Values.web.service.annotations | indent 4 }}
{{- end }}
```

The current version is:
```
{{- if .Values.web.service.annotations }}
  annotations:
    {{ toYaml .Values.web.service.annotations | indent 4 }}
{{- end }}
```
Which does not work because it has too many indentations.

**Comment by hongkongkiwi** [comment]
Fixed with #82

**Comment by Place1** [comment]
Thanks for submitting the PR!
